docs/aca/07-aca-cron-bindings/index.md

```diff
@@ -1,3 +1,7 @@
+---
+canonical_url: https://bitoftech.net/2022/09/09/azure-container-apps-monitoring-and-observability-with-application-insights-part-8/
```

Comment:
this should be: https://bitoftech.net/2022/09/05/azure-container-apps-with-dapr-bindings-building-block/

Reply:
Interesting. It was what you shared in the PR. But anyways i will fix. Thanks for spotting